fga/cache/cachedFgaTenantStore.go

```diff
@@ -0,0 +1,88 @@
+package cache
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"slices"
+	"time"
+
+	"github.com/hashicorp/golang-lru/v2/expirable"
+	openfgav1 "github.com/openfga/api/proto/openfga/v1"
+	"google.golang.org/grpc/status"
+)
+
+type CachedFgaTenantStore struct {
+	cache *expirable.LRU[string, string]
+}
+
+func NewCachedFgaHelper() *CachedFgaTenantStore {
```

Comment:
We have different names `NewCachedFgaHelper` and `NewCachedFgaHelper`.
Lets simplify it to New.
And considering the package name I proposed it will be called store.New()

Reply:
done